cmd/url.go

```diff
@@ -87,8 +89,33 @@ The created URL can be used to access the specified component from outside the O
 			os.Exit(1)
 		}
 
+		componentPorts, err := url.GetComponentServicePortNumbers(client, componentName, applicationName)
+		checkError(err, "unable to get component exposed ports")
+
+		var portFound bool
+
+		if urlPort == -1 {
+			if len(componentPorts) > 1 {
```

Comment:
Can there be a case where a component exposes no ports?
Ex: An internal heartbeat application that checks states of various other components and only updates state in a db and is not exposed to the external?
In which case, throw an error as there's nothing to expose ?
Just thinking...